bootstrap/sql/migrations/native/1.3.0/mysql/schemaChanges.sql

```diff
@@ -36,4 +36,6 @@ where name = 'DataInsightsApplication';
 update metadata_service_entity
 set json = JSON_REPLACE(json, '$.connection.config.secretsManagerProvider', 'db')
 where name = 'OpenMetadata'
-  and JSON_EXTRACT(json, '$.connection.config.secretsManagerProvider') = 'noop';
\ No newline at end of file
+  and JSON_EXTRACT(json, '$.connection.config.secretsManagerProvider') = 'noop';
+
+TRUNCATE TABLE automations_workflow;  
```

Comment:
why are we truncating this table?

Reply:
it is related to the issue we are facing:


```
org.openmetadata.service.exception.SecretsManagerUpdateException: Cannot convert [LinkedHashMap] due to missing converter implementation.
	at org.openmetadata.service.secrets.SecretsManagerUpdateService.updateWorkflow(SecretsManagerUpdateService.java:291)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.openmetadata.service.secrets.SecretsManagerUpdateService.updateWorkflows(SecretsManagerUpdateService.java:111)
	at org.openmetadata.service.secrets.SecretsManagerUpdateService.updateEntities(SecretsManagerUpdateService.java:79)
	at org.openmetadata.service.OpenMetadataApplication.run(OpenMetadataApplication.java:216)
	at org.openmetadata.service.OpenMetadataApplication.run(OpenMetadataApplication.java:121)
	at io.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:67)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:98)
	at io.dropwizard.cli.Cli.run(Cli.java:78)
	at io.dropwizard.Application.run(Application.java:94)
	at org.openmetadata.service.OpenMetadataApplication.main(OpenMetadataApplication.java:506)
```

had a discussion about the same with @pmbrull